index.js

```diff
@@ -88,7 +111,10 @@ exports.handleMessage = function ( hook, context, callback )
 {
   if (context.message.type == 'COLLABROOM' && context.message.data.type == 'RTC_MESSAGE') {
     handleRTCMessage(context.client, context.message.data.payload);
-    callback([null]);
+    callback([null]); // [null]? The docs say null.
```

Comment:
What of this? in etherpad-lite's code, it seems that `callback(null)` (as I did below myself) signals that no other handlers should snatch this message. I didn't see any particular meaning for `[null]`.

Reply:
feck knows, looks like a bug.

Reply:
From testing, it turns out that you can return a single message `callback(x)` or multiple messages `callback([x, y, z])`. (that's JavaScript for you, not sticking to one type) If any of the values is `null` it drops the message after the plugins.